### PR TITLE
specific IL-RFC versions in Interledger readme

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: proposals/interledger/README.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ This data can be used by the website to consult a back-end API to further verify
 
 As `interledger` is an open payment method it will require explicit implementation by browsers. This is an opportunity for browsers to assist in securing the payment by implementing the Payment Pointer protocol in the browser and simply returning the result of this to the Payment Handler.
 
-In this way the browser can perform additional security steps such as filtering the receiver endpoint URL against known malicious origins, ensuring that the certificates used by the receiver endpoint meet strict security standards or enforcing further security measures such as requiring that the origin of the receiver endpoint is the same as (or explicitly trusted by) the origin of the Payment Request API caller.
\ No newline at end of file
+In this way the browser can perform additional security steps such as filtering the receiver endpoint URL against known malicious origins, ensuring that the certificates used by the receiver endpoint meet strict security standards or enforcing further security measures such as requiring that the origin of the receiver endpoint is the same as (or explicitly trusted by) the origin of the Payment Request API caller.
```
Comment (michielbdejong) [inline proposals/interledger/README.md:163]: Not sure why it says this line changed, maybe github's online editor changed the newline character.

**Comment by michielbdejong** [comment]
Motivation: I want to refer to this document and get a specific version of ILP addresses spec and payment pointers spec. If this document itself leaves those versions free, then instead of "web payments with the interledger payment methods" I would have to refer to "web payments with the interledger payment methods and draft-2 ILP addresses and draft-1 payment pointers", which is just very long. :)

**Comment by michielbdejong** [comment]
I linked my github account to [my w3c user](https://www.w3.org/users/97574).

**Comment by adrianhopebailie** [comment]
I don't really agree with this change...

The Web community has become accustomed to "living documents" as specifications, see the WHATWG specs.

Sometimes it's preferable to reference a very specific version of a spec (such as in anther spec) but in a document like this I think it's better to reference the latest version.

**Pull request closed without merging**